client/cluster_test.go

```diff
@@ -124,7 +124,8 @@ func TestAddMachine(t *testing.T) {
 
 	latestImageMaster := api.LatestImage(cloudProvider, api.Master)
 	latestImageWorker := api.LatestImage(cloudProvider, api.Master)
-	firstImageMaster := cloudProvider.MachineImages(api.Master)[0]
+  allImages = cloudProvider.MachineImages(api.Master)
+  firstImageMaster := allImages[ len(allImages) - 2 ]
```

Comment:
To have the same indentation as the other lines.
```suggestion
      allImages = cloudProvider.MachineImages(api.Master)
      firstImageMaster := allImages[ len(allImages) - 2 ]
```

Reply:
Oops... should have checked tabs vs space :smile: